src/context/AppProvider/AppContext.tsx

```diff
@@ -149,6 +156,16 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
             case CHAIN_NAMESPACE.Bitcoin: {
               if (!supportsBTC(wallet)) continue
 
+              if (
+                chainReference === CHAIN_REFERENCE.LitecoinMainnet &&
+                wallet instanceof KeepKeyHDWallet &&
+                isLitecoinEnabled &&
+                versions // without versions, we can't be sure the isLTCSupportedFirmwareVersion is valid
+              ) {
+                // skip litecoin if KK firmware update is required
+                if (!isLTCSupportedFirmwareVersion) continue
```

Comment:
Does product want all or nothing? We would have the option of showing legacy litecoin accounts on old firmware version if we wanted

Reply:
@kaladinlight confirmed per @DiggyDiggy2 we want the "all or nothing" approach of forcing them to update firmware given we waited. partial support can and will cause confusion.